libvast_test/src/events.cpp

```diff
@@ -89,9 +89,11 @@ extract(Reader&& reader, table_slice::size_type slice_size) {
 template <class Reader>
 std::vector<table_slice_ptr>
 inhale(const char* filename, table_slice::size_type slice_size) {
+  caf::settings settings;
+  // A non-positive value disables the time limit.
```

Comment:
Can you add to the comment that we want this because of the weird clock in the test setup?